src/traverse/mod.rs

```diff
@@ -350,7 +350,7 @@ pub(crate) fn extract(
                 if cargo_toml.is_file() {
                     Extraction::Manifest(cargo_toml)
                 } else {
-                    // @todo should we just collect all .rs files here instead?
+                    // TODO should we just collect all .rs files here instead?
```

Comment:
out of curiousity, why use `TODO` instead of `@todo`?

Reply:
It's more common generally, and @ is kind of associated to mentions.
Also there is a nice vs code plugin which adds a pane for all todos and fixmes in a codebase.

Reply:
The plugin is called `Todo Tree`